src/components/Executions/Tables/WorkflowExecutionTable/styles.ts

```diff
@@ -23,11 +24,10 @@ export const useStyles = makeStyles((theme: Theme) => ({
         textAlign: 'right'
     },
     columnActions: {
-        flexGrow: 1,
         flexBasis: workflowExecutionsTableColumnWidths.actions,
         marginLeft: theme.spacing(2),
         marginRight: theme.spacing(2),
-        textAlign: 'left'
+        textAlign: 'right'
```

Comment:
As per our conversation, put actions to right side + updated column nanme size
<img width="1464" alt="Screen Shot 2022-02-02 at 11 19 04 AM" src="https://user-images.githubusercontent.com/55718143/152222248-b7988f3e-9881-4c47-ac88-435e83b38091.png">